### ISSUE TITLE
Question about training hubert models

### ISSUE THREAD

**Issue by GeleiaComPepino** [RVC-Boss/GPT-SoVITS#735]
Hi, I'm trying to train a hubert model in portuguese for use on your project,I have the portuguese dataset (24 hours of dataset), but i tried the fairseq tutorial, and i don't know how to set the params correctly, and pytorch needs to be a LibriSpeech or LibriLight dataset, but i don't have a portuguese of LibriLight, any help with a tutorial?
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/85371842/2e153801-6da7-4171-ac52-1685fd08e34b)
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/85371842/8489ef94-902e-462d-883f-6a969f5ad86a)
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/85371842/8669d520-75d5-4abc-a2cd-e5c7fde051b1)
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/85371842/fcbf147f-6ec2-4c1d-9d46-b9e10e414349)

**Comment by BankNatchapol** [comment]
I don't think you need to train the hubert model. Just use other Portuguese wav2vec model.
